docs/intro.md

```diff
@@ -5,25 +5,37 @@ title: Introduction
 
 ## What is Jitsi?
 
-Jitsi is a [collection of Open Source projects](architecture.md) which provide state-of-the-art video conferencing
+Welcome to Jitsi! A [collection of Open Source projects](architecture.md) which provide state-of-the-art video conferencing
```

Comment:
I would remove the "welcome", because the users are already greeted on the front page: https://jitsi.github.io/handbook/

So the sentence should still begin like before: `Jitsi is a...`